pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -475,6 +475,58 @@ func (adc *attachDetachController) populateDesiredStateOfWorld() error {
 		}
 	}
 
+	// Process Volume-Attachment objects.
+	// If the volume in a VA object is not present in the ASW, add the volume
+	// to ASW as an "uncertain" attachment. If the volume is present in the DSW,
+	// the reconciler will attempt attach on the volume, if the volume is NOT
+	// present in the DSW (could be a dangling attachment), the reconciler will
+	// detach this volume.
+	vas, err := adc.volumeAttachmentLister.List(labels.Everything())
+	if err != nil {
+		klog.Errorf("failed to list VolumeAttachment objects: %v", err)
+		return err
+	}
+	for _, va := range vas {
+		nodeName := types.NodeName(va.Spec.NodeName)
+		pvName := va.Spec.Source.PersistentVolumeName
+		if pvName == nil {
+			klog.V(5).Infof("Skipping the va as its pvName is nil, va.Name: %q, nodeName: %q",
```

Comment:
this should not happen normally, right? maybe make it a warning

Reply:
I see VA might be used for inline volumes for CSI migration feature. I actually not sure whether we need to deal with InlineVolume here. If needed, we can work on it in a separate PR, comment it here and make an issue to track it?

Reply:
Ack. Will add a comment and convert to warning.